src/PHPSemVerChecker/Analyzer/ClassMethodAnalyzer.php

```diff
@@ -109,6 +112,29 @@ public function analyze(Stmt $contextBefore, Stmt $contextAfter)
 					);
 				}
 
+				if ($methodBefore->returnType !== $methodAfter->returnType) {
+					$class = ClassMethodCaseChanged::class;
+					if ($methodBefore->returnType === null) {
```

Comment:
```suggestion
					if ($methodBefore->returnType === null && $methodAfter->returnType !== null) {
```
To match the logic in `Signature::analyze`.